### PR TITLE
fix(tests): switch coverage tool, adjust log_tests

### LINKED ISSUE #1340: test/local/log_tests.js depends on code coverage in order to succeed (which is a fail ;-) )
STR:

1. checkout fxa-auth-server and `npm install`

2. Try any of: 

   - `npm test`
   - `NODE_ENV=dev CORS_ORIGIN=http://foo,http://bar ./scripts/tap-coverage.js test/local test/remote 2>/dev/null`
   - `NO_COVERAGE='' NODE_ENV=dev CORS_ORIGIN=http://foo,http://bar ./scripts/tap-coverage.js test/local test/remote 2>/dev/null`

  Those all pass (they're essentially the same command).

3. Now, without code coverage on:

   - `NO_COVERAGE='1' NODE_ENV=dev CORS_ORIGIN=http://foo,http://bar ./scripts/tap-coverage.js test/local test/remote 2>/dev/null`

   That fails for `test/local/log_tests.js`. 

   Running that test outside of tap-coverage, also fails:

   - `NODE_ENV=dev CORS_ORIGIN=http://foo,http://bar ./node_modules/.bin/tap test/local/log_tests.js 2>/dev/null`
   - `NODE_ENV=dev CORS_ORIGIN=http://foo,http://bar node test/local/log_tests.js 2>/dev/null`
  
   Again, these all fail `test/local/log_tests.js`.

4. Huh? 

/cc @vbudhram, @philbooth 

p.s., go away for a few days, and github switches to an ugly font!

### PR DESCRIPTION
**Pull request by vladikoff**
Fixes #1340

### CODE REVIEW

**Comment by vladikoff** [comment]
@jrgm @vbudhram or @philbooth  r?

File: .travis.yml [hunk 1]
```diff
@@ -4,6 +4,7 @@ language: node_js
 env:
   global:
     - TMPDIR=/tmp
+    - COVERALLS_REPO_TOKEN=vKN3jjhAOwxkv9HG0VBX4EYIlWLPwiJ9d
```
Comment (vbudhram) [inline .travis.yml]: Is there any additional work needed to get it to show up at [coveralls site](https://coveralls.io/github/mozilla/fxa-auth-server)?

**Comment by vbudhram** [comment]
Tested locally and works well, also generates coverage html. Pending question, r+!

File: .travis.yml (hunk 1 above)
Comment (vladikoff) [inline .travis.yml]: Ah I think I need to pass `env` into spawn here: https://github.com/mozilla/fxa-auth-server/pull/1348/files#diff-848030f6d2807bf034c74fb360316d44R15 

updating...

**Pull request merged**